crates/node-maintainer/src/linkers/isolated.rs

```diff
@@ -190,13 +190,13 @@ impl IsolatedLinker {
                                             )
                                         })?;
                                     }
-                                } else if ty.is_dir() {
-                                    async_std::fs::remove_dir_all(&path).await.io_context(|| {
-                                        format!(
-                                            "Failed to delete {} while pruning node_modules.",
-                                            path.display()
-                                        )
-                                    })?;
+                                // } else if ty.is_dir() {
+                                //     async_std::fs::remove_dir_all(&path).await.io_context(|| {
+                                //         format!(
+                                //             "Failed to delete {} while pruning node_modules.",
+                                //             path.display()
+                                //         )
+                                //     })?;
```

Comment:
did you forget to undo this?

Reply:
Yep, I needed to trigger the CI, but I see maybe it's an issue in the main branch https://github.com/orogene/orogene/actions/runs/5296214521/jobs/9587164406